spk/nzbhydra2/Makefile

```diff
@@ -0,0 +1,36 @@
+SPK_NAME = nzbhydra2
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = 1
+SPK_ICON = src/nzbhydra2.png
+
+DEPENDS = cross/$(SPK_NAME)
+SPK_DEPENDS = "Java8>8.0.151:git"
```

Comment:
So if I read the documentation of NBZHydra it seems this installer only supports x64 linux.
https://github.com/theotherp/nzbhydra2
Therefore we should specify this using: 
```
UNSUPPORTED_ARCHS = $ARM_ARCHES $PPC_ARCHES $x86_ARCHES
```

However, since this is a `noarch` package (see few lines below) that wouldn't work since this variable is checked only during compilation (which there is none).
Therefore we need to specify a custom `INFO` file where you can set `archs`, but I don't think this is possible using `spksrc`?
https://developer.synology.com/developer-guide/synology_package/INFO_necessary_fields.html
What do you think @ymartin59?

Reply:
Added